README.md

```diff
@@ -198,6 +199,28 @@ vars:
 vars:
   hubspot__pass_through_all_columns: true # default is false
 ```
+### Adding property label
+For `property_hs_*` columns, you can enable the corresponding, human-readable `property_option`.`label` to be included in the staging models. 
+- **Note** you cannot bring in labels if using `hubspot__pass_through_all_columns: true`.` 
+- To enable labels for a given property, set the property attribute `use_property_label: true`, using the below format.
+
+```yml
+vars:
+  hubspot__ticket_pass_through_columns:
+    - name: "property_hs_fieldname"
+      alias: "fieldname"
+      use_property_label: true
```

Comment:
Same comment from the source that this is not the actual name of the variable being used. It should be  `add_property_label` instead.